content/en/Product Updates/release-notes-september-2024.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Release Notes: September 2024"
+linkTitle: "September 2024"
+weight: 43
+description: >-
+---
+
+{{% pageinfo %}} 
+     Explore What's New from Cobalt This Month
+{{% /pageinfo %}}
+
+### September 2024
+
+#### Updated Pentest Attestation Letters
+
+We revamped the attestation letter's style for better PDF viewing and downloading, aligning it with our current brand guidelines.
+
+{{% pageinfo color="info" %}}
+View more scan controls available [here](https://docs.cobalt.io/platform-deep-dive/scans/scans/).
+{{% /pageinfo %}}
+
+![Attestation Letter Improvements](/release-notes/attestation-letter.png “Attestation “Letter Improvements)
```

Comment:
All 3 images here need to adjust/close the quotation marks at the end around the image description e.g. "Attestation Letter Improvements"